lab-justin/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
This is a good start!  There are a few improvements you can make, see my comments below.